components/alert/examples/00-default.js

```diff
@@ -34,15 +34,17 @@ function Example({ brand }) {
 
 			<h3>Alert heading</h3>
 			<Alert heading="This is a Heading">
-				This alert needs your attention, but it’s not super important. Lorem ipsum dolor sit amet,
-				consectetur adipisicing elit. Quo dolor provident quasi nisi officia tempore fuga autem,
-				animi iste molestiae, qui omnis doloribus aliquid ipsam rem fugiat veniam voluptatem
-				accusamus! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Est, unde quis,
-				molestias nisi quae voluptates nemo quaerat nihil, consequuntur nobis ratione rerum
-				asperiores eveniet dicta maiores quia nostrum. Pariatur, natus. Lorem ipsum dolor sit amet,
-				consectetur adipisicing elit. Fuga, magnam illum harum consequatur, quo sunt impedit quam
-				minus eaque saepe voluptas corrupti voluptatum, sapiente dolor sequi tempore maxime? Neque,
-				obcaecati. <a href="#">Link</a>
+				<p>
```

Comment:
You shouldn't have to do this in the alert components

Reply:
May be used with multiple paragraphs 🤷‍♂️